src/faction_camp.cpp

```diff
@@ -66,6 +67,22 @@ const skill_id skill_archery( "archery" );
 const skill_id skill_swimming( "swimming" );
 
 static const int COMPANION_SORT_POINTS = 12;
+enum class sort_pt_ids : int {
+    ufood, cfood, seeds,
+    weapons, clothing, bionics,
+    tools, wood, trash,
+    books, meds, ammo
+};
+
+static std::array<std::pair<const std::string, point>, COMPANION_SORT_POINTS> sort_point_data = { {
+        { "food for you", point( -3, -1 ) }, { "food for companions", point( 1, 0 ) },
+        { "seeds", point( -1, -1 ) }, { "weapons", point( -1, 1 ) },
+        { "clothing", point( -3, -2 ) }, { "bionics", point( -3, 1 ) },
+        { "all kinds of tools", point( -3, 2 ) }, { "wood of various sorts", point( -5, 2 ) },
+        { "trash and rotting food", point( -6, -3 ) }, { "books", point( -3, 1 ) },
```

Comment:
Looks like text displayed to the user, if so, it should be marked for translation and so on.

Reply:
Oops.  Fixed, thanks.